tool_commands/create_breach_locations/test_breach_location.py

```diff
@@ -227,8 +233,10 @@ class TestBresLocation(unittest.TestCase):
 
     def setUp(self):
         ensure_qgis_app_is_initialized()
-        here = os.path.split(os.path.abspath(__file__))[0]
-        test_db_org = os.path.join(here, "data", "simple_breach_test.sqlite")
+        test_db_org = TEST_DATA_DIR / "simple_breach_test.sqlite"
```

Comment:
Dat `_org`, dat klinkt als "organisation".
Als er "original" mee bedoeld wordt (wat ik vermoed), kun je het dan in `_orig` veranderen? Dat is een meer courante afkorting.